.github/workflows/build-test-e2e.yml

```diff
@@ -25,9 +25,10 @@ jobs:
           retention-days: 1
   playwright-tests:
     needs: [build]
-    runs-on: ${{ github.repository_owner == 'Sage-Bionetworks' && 'ubuntu-22.04-4core-16GBRAM-150GBSSD' || 'macos-latest' }}
+    runs-on: 'macos-latest'
```

Comment:
Use macOS runner for all jobs to decrease flakiness vs linux runners -- see [10x passing runs](https://github.com/hallieswan/SynapseWebClient/actions/runs/7009218290) using macOS runner here.